browser/ui/BUILD.gn

```diff
@@ -548,8 +548,9 @@ source_set("ui") {
     "//brave/components/constants",
     "//brave/components/cosmetic_filters/resources/data:generated_resources",
     "//brave/components/de_amp/common",
-    "//brave/components/debounce/browser",
-    "//brave/components/debounce/common",
+    "//brave/components/debounce/content/browser",
```

Comment:
I don't think we should need `content` here, we only need that for brave_content_browser_client